src/app/Counters/AddDeltaCounter.tsx

```diff
@@ -0,0 +1,65 @@
+import React from 'react';
+import { Button, ButtonVariant, Modal, Text, TextContent, TextInput } from '@patternfly/react-core';
+import { useAddDeltaCounter } from '@app/services/countersHook';
+import { useTranslation } from 'react-i18next';
+
+/**
+ * Delete counter modal
+ */
+const AddDeltaCounter = (props: {
```

Comment:
I would like a test for this, mocking the counter modal if possible. Unit test one I mean

Reply:
Added one unit test, let me know if we need some more cases in that.